watertap/unit_models/translators/translator_asm1_adm1.py

```diff
@@ -184,9 +184,12 @@ def CODd(blk, t):
             doc="COD demand minus readily biodegradable substrate",
         )
         def CODd_step1(blk, t):
-            return (
-                blk.inter_S_S[t]
-                == blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t]
+            return blk.inter_S_S[t] == 0.5 * (
+                (blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t])
+                + sqrt(
+                    (blk.properties_in[t].conc_mass_comp["S_S"] - blk.CODd[t]) ** 2
+                    + eps
+                )
```

Comment:
@agarciadiego I didn't double-check the lit, but is this modification accounted for a shift from the Copp to modified Copp interface?

Reply:
No, its the max(value, 0) formulation

Reply:
Please use the `smooth_max` from IDAES instead: https://github.com/IDAES/idaes-pse/blob/0e90d3e257fe6158c1b7fe79d8bf23c85d561c14/idaes/core/util/math.py#L99